src/string.cr

```diff
@@ -13,74 +13,7 @@ require "c/string"
 # "hello world"
 # ```
 #
-# A backslash can be used to denote some characters inside the string:
-#
-# ```
-# "\"" # double quote
-# "\\" # backslash
-# "\e" # escape
-# "\f" # form feed
-# "\n" # newline
-# "\r" # carriage return
-# "\t" # tab
-# "\v" # vertical tab
-# ```
-#
-# You can use a backslash followed by an *u* and four hexadecimal characters to denote a unicode codepoint written:
-#
-# ```
-# "\u0041" # == "A"
-# ```
-#
-# Or you can use curly braces and specify up to six hexadecimal numbers (0 to 10FFFF):
-#
-# ```
-# "\u{41}" # == "A"
-# ```
-#
-# A string can span multiple lines:
-#
-# ```
-# "hello
-#       world" # same as "hello\n      world"
-# ```
-#
-# Note that in the above example trailing and leading spaces, as well as newlines,
-# end up in the resulting string. To avoid this, you can split a string into multiple lines
-# by joining multiple literals with a backslash:
-#
-# ```
-# "hello " \
-# "world, " \
-# "no newlines" # same as "hello world, no newlines"
-# ```
-#
-# Alternatively, a backslash followed by a newline can be inserted inside the string literal:
-#
-# ```
-# "hello \
-#      world, \
-#      no newlines" # same as "hello world, no newlines"
-# ```
-#
-# In this case, leading whitespace is not included in the resulting string.
-#
-# If you need to write a string that has many double quotes, parentheses, or similar
-# characters, you can use alternative literals:
-#
-# ```
-# # Supports double quotes and nested parentheses
-# %(hello ("world")) # same as "hello (\"world\")"
-#
-# # Supports double quotes and nested brackets
-# %[hello ["world"]] # same as "hello [\"world\"]"
-#
-# # Supports double quotes and nested curlies
-# %{hello {"world"}} # same as "hello {\"world\"}"
-#
-# # Supports double quotes and nested angles
-# %<hello <"world">> # same as "hello <\"world\">"
-# ```
```

Comment:
Please, don't. Having that within the docs, inline is very valuable, especially for the offline version.

Reply:
We should probably include the language reference in the offline docs. But duplicate content is not good.

Reply:
Not having it offline is not good either.

Reply:
Okay, I'll add them back but only for the time being until we figure this out.